invenio_rdm_records/resources/serializers/dublincore/schema.py

```diff
@@ -56,11 +77,11 @@ def get_relations(self, obj):
 
         # Alternate identifiers
         for a in obj["metadata"].get("alternate_identifiers", []):
-            rels.append(f"{a['scheme']}:{a['identifier']}")
```

Comment:
The old logic here would unconditionally append the scheme as a prefix, which looks weird in some cases.
For example, `ark:/123/456` would be turned into `ark:ark:/123/456` which seems redundant to me.

An EAN-13 value of e.g. `9780521425575` would be turned into `ean13:9780521425575` both before and after the PR.